README.md

```diff
@@ -1 +1,90 @@
-This is my virtual pet project, completed as part of my studies with Command Shift.
+# Virtual Pet Project
+
+## Description
+This is my virtual pet project, created during my studies of JavaScript with Command Shift's bootcamp. The project was an exercise in test-driven-development. To build the pet I had to write unit tests, and then write functions which passed these tests. The project is intended to be run in the Node REPL, which is where I feature tested it. You will therefore need to have node.js installed on your machine to run this project. The virtual pet can be fed, exercised, grow up and have children. The pet can also die, so be careful!
+
+## How-to
+1. Download the project to your machine, and navigate to the ./virtualpet directory in your command line.
+
+2. Open the Node REPL - { $node }
+
+3. Initalize the pet constructor function. { const pet = require('./src/pet'); }
+
+4. Now you can create and name your pet. { const Fido = new pet('Fido'); }
+
+5. Your pet's initial stats are as follows: 
+
+    name: 'Fido',
+    
+    age: 0,
+    
+    hunger: 0,
+    
+    fitness: 10,
+    
+    children: []
```

Comment:
It would be a good idea to look into markdown language for your readmes, this is the industry standard tool for this. This could be updated to use code blocks for example:

```
## Installation

Clone the project to your machine, navigate to the new directory and install the project:

```
git clone git@github.com:JordsCodes/virtual-pet.git
cd virtual pet
npm i
```

Open the Node REPL:
```
node
```

Initalize the pet constructor function
```
const pet = require('./src/pet');
```

Now you can create and name your pet.
```
const Fido = new pet('Fido');
```

Your pet's initial stats are as follows: 
```
name: 'Fido',
age: 0,
hunger: 0,
fitness: 10,
children: []
```
```